tests/main/lxd/task.yaml

```diff
@@ -0,0 +1,64 @@
+summary: Ensure that lxd works
+
+# only run this on ubuntu, we need a reasonable new kernel
+systems: [-debian-*]
```

Comment:
You need to tweak this to exclude fedora and openSUSE though ideally only after checking it really doesn't work there